pkg/build/util/util.go

```diff
@@ -74,3 +75,9 @@ func NameFromImageStream(namespace string, ref *kapi.ObjectReference, tag string
 func IsBuildComplete(build *buildapi.Build) bool {
 	return build.Status.Phase != buildapi.BuildPhaseRunning && build.Status.Phase != buildapi.BuildPhasePending && build.Status.Phase != buildapi.BuildPhaseNew
 }
+
+// ConfigSelector returns a label Selector which can be used to find all
+// builds for a BuildConfig.
+func ConfigSelector(name string) labels.Selector {
+	return labels.Set{buildapi.BuildConfigLabel: name}.AsSelector()
```

Comment:
> @kargakis existing code needs to check for both labels.

I cannot see how this can be done w/o an OR selector... Older builds won't be picked up by newer versions of controllers/clients if you require both labels.